admin/lint_json.m

```diff
@@ -0,0 +1,52 @@
+function lint_json(filesin, outputfile)
+    if isempty(filesin)
+        % Default to glob all
+        filesin = "**/*.m";
+    end
+    if isempty(outputfile) % Default to stdout
+        fh = 1;
+    else % Open file
+        fh = fopen(outputfile,'w');
+        if fh == -1
+            error("MATLAB:FileOpenError", "Failed to open file %s", outputfile);
+        end
+        cleanup = onCleanup(@()(fclose(fh)));
+    end
+
+
+    files = [];
+    for i = 1:numel(filesin)
+        flist = dir(filesin{i});
```

Comment:
Trying to run this file, I get a `Brace indexing is not supported for variables of this type.` error. On line 4 `filesin` is defined as a string, which is not `{}` indexable